<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rotate Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    color: #b1b1b1;
    background-color: #323232;
    selection-background-color:#323232;
    selection-color: black;
    background-clip: border;
    border-image: none;
    border: 0px transparent black;
    outline: 0;
}

QWidget:item:hover
{
    background-color: #D1DBCB;
    color: black;
}

QWidget:item:selected
{
    background-color: #D1DBCB;
    border: 0px
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="do_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font-weight: bold;

border-radius: 15px;
color: #1DA1F2;
}</string>
        </property>
        <property name="text">
         <string>До</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="first_image_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="after_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font-weight: bold;

border-radius: 15px;
color: #1DA1F2;
}</string>
        </property>
        <property name="text">
         <string>После</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="itog_image_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="url_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
font-weight: bold;

border-radius: 15px;
color: #1DA1F2;
}</string>
          </property>
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="url_line_edit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
  background-color: #19232D;
  padding-top: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-bottom: 2px;
  /* This QLineEdit fix  103, 111 */
  padding-left: 4px;
  padding-right: 4px;
  border-style: solid;
  border: 1px solid #32414B;
  color: #F0F0F0;
  border-radius: 15px;
}

QLineEdit:disabled {
  background-color: #19232D;
  color: #787878;
}

QLineEdit:hover {
  border: 1px solid #148CD2;
  color: #F0F0F0;
}

QLineEdit:focus {
  border: 1px solid #1464A0;
}

QLineEdit:selected {
  background-color: #1464A0;
  color: #32414B;
}</string>
          </property>
          <property name="placeholderText">
           <string>Здесь должен быть url фотографии(png,jpg)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="upload_photo_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Нажмите на кнопку, чтобы загрузить фотографию на Ваш компьютер и в приложение&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;

}
QPushButton:hover{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #fff;
}
QPushButton:pressed{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #cccccc;
}
QPushButton:disabled {
  background-color: #32414B;
  color: #787878;
  padding: 3px;
}
 QToolTip {
     border: 2px solid darkkhaki;
     padding: 5px;
     border-radius: 3px;
     opacity: 200;
 }</string>
        </property>
        <property name="text">
         <string>Загрузить изображение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rotate_90_left_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Нажмите на кнопку, чтобы повернуть изображение на 90° влево&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;

}
QPushButton:hover{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #fff;
}
QPushButton:pressed{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #cccccc;
}
QPushButton:disabled {
  background-color: #32414B;
  color: #787878;
  padding: 3px;
}

 QToolTip {
     border: 2px solid darkkhaki;
     padding: 5px;
     border-radius: 3px;
     opacity: 200;
 }</string>
        </property>
        <property name="text">
         <string>Повернуть на 90° влево</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rotate_90_right_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Нажмите на кнопку, чтобы повернуть изображение на 90° вправо&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;

}
QPushButton:hover{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #fff;
}
QPushButton:pressed{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #cccccc;
}
QPushButton:disabled {
  background-color: #32414B;
  color: #787878;
  padding: 3px;
}
 QToolTip {
     border: 2px solid darkkhaki;
     padding: 5px;
     border-radius: 3px;
     opacity: 200;
 }</string>
        </property>
        <property name="text">
         <string>Повернуть на 90° вправо</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rotate_180_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Нажмите на кнопку, чтобы повернуть изображение на 180°&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;

}
QPushButton:hover{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #fff;
}
QPushButton:pressed{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #cccccc;
}
QPushButton:disabled {
  background-color: #32414B;
  color: #787878;
  padding: 3px;
}
 QToolTip {
     border: 2px solid darkkhaki;
     padding: 5px;
     border-radius: 3px;
     opacity: 200;
 }</string>
        </property>
        <property name="text">
         <string>Повернуть на 180°</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Нажмите на кнопку, чтобы перейти на окно назад&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;

}
QPushButton:hover{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #fff;
}
QPushButton:pressed{
font-weight: bold;
border: 1px solid #1DA1F2;
border-radius: 15px;
color: #1DA1F2;
background-color: #cccccc;
}
QPushButton:disabled {
  background-color: #32414B;
  color: #787878;
  padding: 3px;
}
 QToolTip {
     border: 2px solid darkkhaki;
     padding: 5px;
     border-radius: 3px;
     opacity: 200;
 }</string>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
